### PR TITLE
Issues#83

### PR DESCRIPTION
**Pull request by Gvozd**
I try to fix problem from https://github.com/NV/CSSOM/issues/83
Also, i fix exists tests

### CODE REVIEW

File: spec/parse.spec.js [hunk 1]
```diff
@@ -591,10 +591,20 @@ var TESTS = [
...
+						parentRule: null,
+						media: ["all"],
+						cssRules: {}
+					}
+				],
+				parentStyleSheet: null
+			};
+			result.cssRules[0].parentStyleSheet = result;
+			return result;
+		})()
```
Comment (NV) [inline spec/parse.spec.js]: `@mediall` is not a valid at-rule. The test was correct — the implementation was wrong.

File: lib/parse.js [hunk 2]
```diff
@@ -32,7 +32,8 @@ CSSOM.parse = function parse(token) {
 		"importRule-begin": true,
 		"importRule": true,
 		"atBlock": true,
-		'documentRule-begin': true
+		'documentRule-begin': true,
+		"value-parenthesis": true
```
Comment (NV) [inline lib/parse.js]: This isn't used anywhere.

**Comment by Gvozd** [comment]
@NV , please look new variant

File: lib/parse.js (hunk 2 above)
Comment (Gvozd) [inline lib/parse.js]: this used in test ".gradient{background: -moz-linear-gradient(/*);*/top, #1E5799 0%, #7db9e8 100%)}" 
I think, this is correct for this test

File: spec/parse.spec.js (hunk 1 above)
Comment (Gvozd) [inline spec/parse.spec.js]: I reverted test, and fix normally

File: lib/parse.js (hunk 2 above)
Comment (NV) [inline lib/parse.js]: Oh, I see.

**Comment by Gvozd** [comment]
bump
i revert test with @mediall, and fix normally
please look, and merge, or comment
PS Are you speak russian?

**Comment by Gvozd** [comment]
bump

**Pull request closed without merging**